### PR TITLE
Pie trace cleaning

### PR DESCRIPTION
**Pull request by mdtusz**
Similar to the box trace cleanup.

### CODE REVIEW

File: src/traces/pie/attributes.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * LICENSE file in the root directory of this source tree.
 */
 
+'use strict';
```
Comment (etpinard) [inline src/traces/pie/attributes.js]: there's no functions in this, so no need for `'use strict';`

That said, I don't mind making `'use strict'` for all files even to one without functions.

File: src/traces/pie/style_one.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Color = require('../../components/color');
+
+module.exports = function styleOne(s, pt, trace) {
```
Comment (etpinard) [inline src/traces/pie/style_one.js]: Nicely done. This gets used in the legend code.

File: src/traces/pie/attributes.js (hunk 1 above)
Comment (etpinard) [inline src/traces/pie/attributes.js]: @bpostlethwaite what's your take on :arrow_double_up: ?

Comment (bpostlethwaite) [inline src/traces/pie/attributes.js]: I am inclined to put it everywhere we don't have to think about it

Comment (etpinard) [inline src/traces/pie/attributes.js]: @bpostlethwaite :beers: thanks

**Comment by etpinard** [comment]
:dancer:

**Pull request merged**